tests/manage/test_run_io_multiple_pods.py

```diff
@@ -35,7 +35,7 @@ class BaseRunIOMultiplePods(ManageTest):
     """
     Run IO on multiple pods in parallel
     """
-    num_of_pvcs = 10
+    num_of_pvcs = 1
```

Comment:
Why is this change in this pull request? I'm just curious.

Reply:
I'm asking because I would rather prevent stashing unrelated changes into single PR.

Reply:
left it changed by mistake, reverted